### PR TITLE
Update types of wallets section of wallets page

### PR DESCRIPTION
**Pull request by corwintines**
## Description
- Update title `Types of wallet` to `Types of wallet interfaces`
    - Context on this, thought it was worth differentiating that these are the types of ways to interface with an account. Types of wallets to me is more Private key, smart contract, MPC. Not necessarily the device which this is providing information on.
- Add a short description
- Add Brower Extensions as a type of wallet interface.

### CODE REVIEW

File: src/intl/en/page-wallets.json [hunk 1]
```diff
@@ -53,8 +53,10 @@
   "page-wallets-try-dapps": "Try some dapps",
   "page-wallets-try-dapps-alt": "An illustration of Ethereum community members working together",
   "page-wallets-try-dapps-desc": "Dapps are applications built on Ethereum. They’re cheaper, fairer and kinder on your data than most traditional applications.",
-  "page-wallets-types": "Types of wallet",
-  "page-wallets-web-browser": "Web wallets that let you interact with your account via a web browser",
+  "page-wallets-types": "Types of wallet interfaces",
```
Comment (samajammin) [inline src/intl/en/page-wallets.json:56]: Hmmm so what is a "wallet" vs. what is a "wallet interface"? This now confuses me a bit.

Comment (samajammin) [reply src/intl/en/page-wallets.json:56]: To me these are still "types of wallets". Not convinced we should introduce yet more jargon.

File: src/intl/en/page-wallets.json [hunk 2]
```diff
@@ -53,8 +53,10 @@
...
+  "page-wallets-types-desc": "There are a few ways to interface and interact with your account:",
+  "page-wallets-web-browser": "Browser wallets let you interact with your account directly in the browser",
```
Comment (samajammin) [inline src/intl/en/page-wallets.json:58]:
```suggestion
  "page-wallets-web-browser": "Browser wallets are web applications that let you interact with your account directly in the browser",
```

File: src/intl/en/page-wallets.json (hunk 1 above)
Comment (samajammin) [inline src/intl/en/page-wallets.json:56]:
```suggestion
  "page-wallets-types": "Types of wallets",
```

**Review by samajammin — CHANGES_REQUESTED** [review]
Suggested a couple changes. Curious what other folks think here.

**Comment by konopkja** [comment]
> ## Description
> * Update title `Types of wallet` to `Types of wallet interfaces`
>   
>   * Context on this, thought it was worth differentiating that these are the types of ways to interface with an account. Types of wallets to me is more Private key, smart contract, MPC. Not necessarily the device which this is providing information on.
> * Add a short description
> * Add Brower Extensions as a type of wallet interface.

this assumes others also have this background knowledge about smart contract, private key etc. I agree with Sam here, that this level of detail maybe unnecessary to differentiate.

**Comment by corwintines** [comment]
Thanks for the input @samajammin @konopkja . Brought in the change requests. I think this should be good now!

**Review by samajammin — APPROVED** [review]

**Pull request merged**